configs/openmrs/initializer_config/concepts/voidBahmniDispositionConfigs.csv

```diff
@@ -0,0 +1,9 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Answers,Members,_version:1,_order:450
+efaf4399-601c-4479-aba2-21591a6eccae,TRUE,org.openmrs.module.emrapi:TRANSFER,Transfer Patient,Transfer Patient,Transfer Patient,Transfer Patient,,,Misc,N/A,,,,
```

Comment:
Do the Bahmni concepts have the emrapi mappings?
I don't think so, right? Maybe just remove that.

Reply:
What I understand by Bahmni mapping is the ones that I have deleted from here 
![Uploading Screenshot 2025-02-24 at 3.53.19 PM.png…]()